Cargo.toml

```diff
@@ -18,6 +18,7 @@ varnish-sys = { path = "./varnish-sys", version = "=0.0.19" }
 #
 # These dependencies are used by one or more crates, and easier to maintain in one place.
 bindgen = "0.70.1"
+glob = "0.3.1"
```

Comment:
do we also need it here? Asking as you only seem to be using it in `varnish/`

Reply:
many of these are only used once per some crate.  The goal here is to have all "versioned" crates in one place - so we don't need to look at multiple places to do the version updates -- our crates simply point to this one list.  We also only have one lock file. Also, if one of the dependencies starts being needed in more than one crate, we don't need to move things around - just add another `foo.workspace = true`.

Reply:
P.S. the macros PR adds like 10 more to this list btw